modules/optimizing-energy-use/slides_optimizing_energy_use.md

```diff
@@ -53,33 +53,28 @@ Note:
 
 <!-- .slide: data-state="standard" -->
 
-## Right tool for each task
+### Right tool for each task
 
 Are there slow programming languages that should be
 avoided ?
- - each tool has it's purpose.
+ - each tool has its purpose.
 Python: great for stitching pieces together, easy prototyping
 C/C++/Fortran: better at crunching number fast, but not
 so good at doing string manipulation, ...
 
-Benchmark available in the litterature can be missleading.
+Benchmarks available in the literature can be missleading.
 
 -> Use the right tool for each task (could simply be the 
```

Comment:
Alright, but where do I find a guide to what is the right tool for each task?

Reply:
Yes, fair enough. I have reworded the slide a bit here, since the way to choose can be quite problem specific. I have changed the slide to say that, with python/R etc, that is fine but it matters more whether the libraries you are using are efficient.